pyanaconda/exception.py

```diff
@@ -321,7 +321,9 @@ def initExceptionHandling(anaconda):
 def lsblk_callback():
     """Callback to get info about block devices."""
 
-    return util.execWithCapture("lsblk", ["--perms", "--fs", "--bytes"])
+    options = "NAME,SIZE,OWNER,GROUP,MODE,FSTYPE,LABEL,UUID,PARTUUID,MOUNTPOINT"
```

Comment:
Hi, as I'm comparing the outputs, it lacks the columns FSAVAIL and FSUSE% that are in the output of the previous lsblk incantation. They are apparently added by the `--fs` option, even if manpage says nothing about these.

```suggestion
    options = "NAME,SIZE,OWNER,GROUP,MODE,FSTYPE,LABEL,UUID,PARTUUID,FSAVAIL,FSUSE%,MOUNTPOINT"
```